plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -534,14 +547,19 @@ public String getBindExpression()
             }
 
             @Override
-            public void set(PreparedStatement statement, int index, long utcMillis)
-                    throws SQLException
+            public void set(PreparedStatement statement, int index, long utcMillis) throws SQLException
```

Comment:
Not sure, but are these all meant to be on newlines?

Reply:
Makes sense to keep it consistent here & elsewhere